nntrainer/tensor/tensor.h

```diff
@@ -1680,6 +1680,13 @@ class Tensor {
    */
   Tensor clone() const;
 
+  /**
+   * @brief     Convient wrapper for inplace copy of @a this.
```

Comment:
```suggestion
   * @brief     Convenient wrapper for inplace copy of @a this.
```
Is it typo? Do you mean convenient ?